protocol/contracts/lease/src/lease/repay.rs

```diff
@@ -1,18 +1,18 @@
-use currencies::PaymentGroup;
 use sdk::cosmwasm_std::Timestamp;
```

Comment:
since all cosmos-sdk types, functions, etc are exposed from the Nolus package named `sdk`, we order their `use` clauses as if they are local for the Nolus, not in the section for the external for the Nolus packages